compose/ogmios/Dockerfile

```diff
@@ -81,4 +81,5 @@ EXPOSE 1337/tcp 3000/tcp 12788/tcp 12798/tcp
 HEALTHCHECK --interval=10s --timeout=5s --retries=1 CMD /bin/ogmios health-check
 STOPSIGNAL SIGINT
 COPY --from=haskell-builder /app/ogmios-src/scripts/cardano-node-ogmios.sh cardano-node-ogmios.sh
+RUN ln -s /sbin/tini /tini
```

Comment:
@michalrus FYI

Reply:
I don’t know if this is needed if later, in `ENTRYPOINT`, we call `tiny` (from `PATH`) instead of `/tiny` (absolute/relative), but it doesn’t hurt =)

The original Dockerfile calls `/tiny`

Reply:
Yep, later we have [this override](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/e2e/docker-compose.yml#L43).

Since
> The original Dockerfile calls `/tiny`

I preferred to make our custom image as more compliant to the original as possible rather than applying a workaround the the override to handle the slight difference fixed by this change.

Reply:
Ah, you’re right! Sorry, I missed it 🫣